src/app/event-card/event-card.component.scss

```diff
@@ -0,0 +1,176 @@
+$primary-color: #084778;
```

Comment:
when you pull the latest changes from the master branch you will notice that there are CSS variables in the styles.scss file which can be used by `var(--color)`. So some of these colors might be repeated.